packages/ui/src/graph-view/graph.module.css

```
@@ -0,0 +1,76 @@
+}
+
+.graph .globalGraphIcon:hover {
+  background-color: var(--lightgray, #e0e0e0);
+}
+
+.graphContainer {
+  width: 100%;
+  height: 100%;
+}
+
+:global(.global-graph-outer) {
```

Stylelint (17.10.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)